src/SFA.DAS.EmployerIncentives.Web/Services/Apprentices/ApprenticesService.cs

```diff
@@ -20,23 +21,88 @@ public ApprenticesService(HttpClient client, IHashingService hashingService)
             _hashingService = hashingService;
         }
 
-        public async Task<IEnumerable<ApprenticeshipModel>> Get(ApprenticesQuery query)
+        public async Task<EligibleApprenticeshipsModel> Get(ApprenticesQuery query)
```

Comment:
Would the complexity in this method not be better off within the outer api?

Reply:
The problem we're facing at the moment is that the call to the APIM to fetch apprentices times out in larger volumes, because the APIM call fetches the apprentices from the Commitments API and then makes calls to the EI API to check the ULN hasn't been applied for previously.
If this logic is in the APIM layer, there is a danger that the call will time out, particularly for larger employers where they have previously applied for some of their apprentices.

Reply:
Is that a greater risk than the UI timing out?  Ultimately, the same number of calls between the outer & inner APIs would occur.

Reply:
Greater risk of one request from the web to APIM (which results in hundreds of calls to Commitments and EI APIs) timing out than one request that results in a maximum of 50 (page size) calls

Reply:
But the APIM request should only return the max page size so the same number of backend calls are required?